### PR TITLE
refactor: Header component

### PR DESCRIPTION
**Pull request by ShrimpCryptid**
Problem
=======
Refactors the header into a reusable series of components as part of #250.

*Estimated review size: small, <15 minutes. (Recommend turning off whitespace)*

Solution
========
- Moves the header bar into its own `Header.tsx` component file, which pages can reuse.
  - We can't mount the header as part of the root component, because many of the buttons that go in the header (export, copy, etc.) are context-specific and require access to page state.
  - This component will eventually be used by the landing page.

## Type of change
* New feature (non-breaking change which adds functionality)

Steps to Verify:
----------------
1. Open the preview link and confirm that the inputs and interactions in the header work: https://allen-cell-animated.github.io/nucmorph-colorizer/pr-preview/pr-257/
2. Open the error page. The header should also appear: https://allen-cell-animated.github.io/nucmorph-colorizer/pr-preview/pr-257/#/bad

Screenshots (optional):
-----------------------
image

### CODE REVIEW

File: src/assets/AICS-logo-and-name.svg
Comment (ShrimpCryptid) [inline src/assets/AICS-logo-and-name.svg:1]: Edited the SVG's title attribute to say "Allen Institute for Cell Science"

File: src/Viewer.module.css
Comment (ShrimpCryptid) [inline src/Viewer.module.css:1]: Moved to `Header.tsx`

**Comment by meganrm** [comment]
Design feedback: it seems weird to have the help be a dropdown that only has one item. is that because you plan to have more in that dropdown in the future?

**Review by meganrm — APPROVED** [review]

**Comment by ShrimpCryptid** [comment]
> the preview doesn't load the dataset (I get an error) is that expected?

It should be loading... Are you on VPN?

**Comment by ShrimpCryptid** [comment]
> Design feedback: it seems weird to have the help be a dropdown that only has one item. is that because you plan to have more in that dropdown in the future?

Yup! There are a few issues in the backlog for adding additional options to the help dropdown. See #241  and #239.

**Review by frasercl — APPROVED** [review]
Looks good!

**Pull request merged**